service/currency/arch-go.yaml

```diff
@@ -0,0 +1,42 @@
+version: 1
+threshold:
+  compliance: 100
+  coverage: 40
+dependenciesRules:
+  - package: "**.internal/clients.*"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.internal/config.**"
+        - "**.internal/entities.**"
+        - "**.internal/service.**"
+        - "**.internal/parser.**"
+
+  - package: "**.internal/entities.**"
+    shouldNotDependsOn: 
+      internal: 
+        - "**.internal/clients.**"
+        - "**.internal/config.**"
+        - "**.internal/grpc.**"
+        - "**.internal/mocks.**"
+        - "**.internal/service"
+        - "**.internal/parser.**"
```

Comment:
Is it possible to say that entity shouldn't depend on anything?

Reply:
Rewrited arch tests for entities package so it can depend only on itself instead of forbiding to depend on other packages (shouldOnlyDependsOn -> shouldNotDependsOn)